Makefile

```diff
@@ -9,66 +9,73 @@
 	mkdir -p work/ILSVRC
 	mkdir work/alov
 
-	touch .data_folder_structure_sentinel
+	touch work/.data_folder_structure_sentinel
 
 ####################
-# ALOV300++ Datset # 
+# ALOV300++ Datset #
 ####################
 
-# Frames # 
-data/alov/frames/alov300++_frames.zip: 
+# Frames #
+data/alov/frames/alov300++_frames.zip:
 	curl http://isis-data.science.uva.nl/alov/alov300++_frames.zip \
 		-o data/alov/frames/alov300++_frames.zip
-	
-	tar xvf data/alov/frames/alov300++_frames.zip 
+
+	tar xvf data/alov/frames/alov300++_frames.zip
 
 	mv imagedata++ data/alov/frames/
 
-# Bounding Boxes # 
-data/alov/bb/alov300++GT_txtFiles.zip: 
+# Bounding Boxes #
+data/alov/bb/alov300++GT_txtFiles.zip:
 	curl http://isis-data.science.uva.nl/alov/alov300++GT_txtFiles.zip \
 		-o data/alov/bb/alov300++GT_txtFiles.zip
 
-	unzip data/alov/bb/alov300++GT_txtFiles.zip 
+	unzip data/alov/bb/alov300++GT_txtFiles.zip
 
 	mv alov300++_rectangleAnnotation_full/ data/alov/bb/bb
 
 alov: data/alov/frames/alov300++_frames.zip \
-	data/alov/bb/alov300++GT_txtFiles.zip 
+	data/alov/bb/alov300++GT_txtFiles.zip
 
 ##############
-# ILSVRC2014 # 
+# ILSVRC2014 #
 ##############
 
-# Training Bounding Boxes # 
-data/ILSVRC/bb/ILSVRC2014_DET_bbox_train.tgz: 
+# Training Bounding Boxes #
+work/.imagenet_training_bbox_sentinel:
```

Comment:
We're now getting rid of the tgz files as discussed in gitter and using sentinels. Sentinels are created at very last step, so that if something errors (e.g. tar xvf fails), we don't create a misleading sentinel suggesting the operation was completed.